mode/sql/sql.js

```diff
@@ -288,11 +288,11 @@ CodeMirror.defineMode("sql", function(config, parserConfig) {
 
   CodeMirror.defineMIME("text/x-mssql", {
     name: "sql",
-    client: set("charset clear connect edit ego exit go help nopager notee nowarning pager print prompt quit rehash source status system tee"),
-    keywords: set(sqlKeywords + "begin trigger proc view index for add constraint key primary foreign collate clustered nonclustered declare exec"),
+    client: set("$partition binary_checksum checksum connectionproperty context_info current_request_id error_line error_message error_number error_procedure error_severity error_state formatmessage get_filestream_transaction_context getansinull host_id host_name isnull isnumeric min_active_rowversion newid newsequentialid rowcount_big xact_state object_id"),
```

Comment:
You appear to have dropped most of the original client keywords here. Is that intentional?

Reply:
I replaced client keywords with these [keywords](https://docs.microsoft.com/en-us/sql/t-sql/functions/system-functions-transact-sql).